core/org.wso2.carbon.user.api/src/main/java/org/wso2/carbon/user/api/ClaimManager.java

```diff
@@ -22,16 +22,15 @@
  * <p/>
  * A claim is a name-value pair
  */
-public interface ClaimManager {
+public interface  ClaimManager {
 
     /**
-     * Retrieves the attribute name of the claim URI.
-     *
-     * @param claimURI The claim URI
+     * @param domainName
+     * @param claimURI
      * @return
      * @throws UserStoreException
      */
-    String getAttributeName(String claimURI) throws UserStoreException;
+    public String getAttributeName(String domainName, String claimURI) throws UserStoreException;
```

Comment:
In an interface, all methods are public and abstract. Therefore, one can consider using access modifiers as redundant.